UN-OCHA/gho-2022-site
# GHO-214: add the Article Card lists

# GHO-214

The homepage for this year has decided to adopt the "card list" style that GHI comps reliev heavily upon. This branch adds some Paragraph types, CD subtheme components, and removes a few Homepage customizations in order to tie it all together. Sample screenshot from Firefox below. It demonstrates the "show first row as 2-column" option in the Paragraph, and a second section without the 2-col treatment.

## Testing

1. It's easiest to grab a snapshot of the dev DB first. Doing so will give you a collection of fully-written articles to choose from for the actual setup.
2. Edit the Homepage and in the Paragraphs field add the following:
  - Heading paragraph
  - Text paragraph
  - Article Card List paragraph, and choose 5 nodes. Check the box to make the first row 2-col.
  - Heading paragraph, pre-title should be "Chapter One"
  - Text paragraph
  - Article Card List, and choose 6 nodes. Leave box unchecked.
3. Save node and view the page

![GHO-214-testing-result](https://user-images.githubusercontent.com/254753/140338829-adeb0bb3-2c58-4265-86a6-c13f6283a9bb.png)
